wiki/Ranking_Criteria/en.md

```diff
@@ -69,24 +81,27 @@ Glossary
+-   **Excluding a mapset’s hardest difficulty, a difficulty’s name must accurately indicate its level of difficulty.** Conventional difficulty names vary between modes, but any set of clearly progressive difficulty names can be alternatively used. Additionally, a mapset’s hardest difficulty should not use a name misrepresentative of its difficulty.
+-   **A difficulty’s name must be unrelated to a username.** Guest difficulties, however, may indicate possession with its mappers’ username or nickname. (e.g. Guest Mapper’s Insane). Words that happen to be usernames are acceptable within difficulty names as long as they relate to the song.
+-   **Additionally, a mapset host cannot indicate possession in a difficulty’s name.** (e.g. Mapset Host’s Insane). Conflicts caused by mapping multiple songs with the same metadata are the only exception.
+-   **A mapset host must have equal or more drain time mapped than any guest difficulty mappers.** This is to provide credit where credit is due.
+-   **A mapset host and guest mappers can make changes to their respective difficulties as they wish.** If there is a disagreement between both, the mapset host must delete the guest contribution upon request. If a guest mapper cannot be contacted for a month, they will be assumed to agree with any changes. 
 
 ### Guidelines
 
--   **The difficulty spread should be linear and reasonable.** Linear difficulty spread means your difficulties have a comparable gap in star rating between each other. A reasonable spread means the usage of [difficulty appropriate gameplay elements](/wiki/Difficulties). If your mapset does not have an Easy difficulty your Normal difficulty should follow the general guidelines for Easy difficulties. If your Insane difficulty is at or above a 5 star rating it is recommended to include another Insane level difficulty between Hard and the 5+ Insane.
--   **Your difficulties should all end at the same spot.** Having a fully-mapped Normal/Hard and a half-mapped Easy just looks sloppy/lazy. A full Easy may look boring to you, but not to a player that can't handle the harder difficulties.
+-   **Avoid incomprehensible username combinations to indicate possession of a collaborative guest difficulty.** If it’s unclear whose usernames are combined, simplification is recommended.
+-   **Avoid difficulty names with descriptive elements not clearly related to a guest difficulty mapper or a level of difficulty.** (e.g. Mapper’s Tragic Love Extra)
+-   **Usernames indicating possession of a guest difficulty should be consistent between multiple mapsets.** Varying nicknames for one user makes interpreting who created a difficulty ambiguous or misleading.
+-   **Avoid unicode characters in a difficulty’s name.** These can cause errors with the beatmap submission system and problems for certain users when appearing in chat.
 
 ## Metadata
 
```

Comment:
"incorrectly" sounds a little more formal than "wrongly"
It's in the section below, not sure how to comment specifically for that part

Reply:
can't see what this is referencing